pkg/apis/rook.io/v1/types.go

```diff
@@ -161,6 +161,15 @@ type LabelsSpec map[KeyType]Labels
 // Labels are label for a given daemons
 type Labels map[string]string
 
+type GrowthPolicy struct {
+	// maxsize for the capacity of DeviceStorage
+	// Optional
+	Maxsize string `json:"maxsize,omitempty"`
```

Comment:
Godocs should document the go field names following the convention of the rest of the doc.
The written equivalent is "max size" (2 words) and thus should be in camelCase.
Descriptions should provide fuller instructions for users.

```suggestion
	// MaxSize defines the maximum size a Volume Claim is allowed to grow to.
	// Optional
	MaxSize string `json:"maxSize,omitempty"`
```